### PR TITLE
Fix - ADF-662 Stabilize fileSelector and filePreviewer

### PR DESCRIPTION
**Pull request by tereshhhchenko**
Jira https://oat-sa.atlassian.net/browse/ADF-662 

Related PR https://github.com/oat-sa/tao-core-ui-fe/pull/355

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "1.15.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.34.1",
+        "@oat-sa/tao-core-ui": "git+https://github.com/oat-sa/tao-core-ui-fe.git#fix/ADF-627/hasClass-is-not-a-function",
```
Comment (habemuscode) [inline views/package.json:21]: Reminder to don't forget to change this

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-shpran.playground.kitchen.it.taocloud.org:41481/

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-shpran.playground.kitchen.it.taocloud.org:41481/

**Pull request merged**